### DISCUSSION TITLE
TechEmpower Benchmarks

### DISCUSSION THREAD

**Discussion by dustinmoris** [panjf2000/gnet#212] [General]
Hi,

Is there an appetite to try and push gnet even higher in the [benchmarks](https://www.techempower.com/benchmarks/#section=data-r20&hw=ph&test=plaintext)?

The current margins in the Plaintext category are so small that a few tweaks could possibly push gnet to the very top:

<img width="1191" alt="image" src="https://user-images.githubusercontent.com/4580714/119841572-d6855e00-befd-11eb-838a-a216afd57ad3.png">

Also would be nice to participate in the [Fortunes](https://www.techempower.com/benchmarks/#section=data-r20&hw=ph&test=fortune) category which is really the Gold standard of benchmarks as well as JSON serialization perhaps.

Is this something you'd be interested in?

**Comment by panjf2000** [comment]
Plaintext is the simplest benchmark for gnet in techempower, I think  Fortunes is also fine to do but I don't have much time for it these days, I'll think about it when I have time later.
